root/etc/e-smith/templates/etc/cron.d/nethserver-blacklist/10base

```diff
@@ -14,4 +17,12 @@
     } else {
         $OUT .= "# Blacklists (dns) are disabled"
     }
+
+    $OUT .= "\n";
+
+    if (($ip_status eq 'enabled') && ($geoipStatus eq 'enabled')) {
+        $OUT .= '* * 1 * * root sleep $(( ( RANDOM \% 7200 ) )); /usr/share/nethserver-blacklist/download geoip'
```

Comment:
GeoIP lists are updated quite often, so I would download it once a day.

Reply:
done